### ISSUE TITLE
Cannot Find Java Version..

### ISSUE THREAD

**Issue by theuserscull** [MultiMC/MultiMC5#1799]
System Information
-----------------------------
MultiMC version: 5

Operating System: Windows 10 64bit

Summary of the issue or suggestion:
----------------------------------------------
Just the norm of me, trying to entertain my self. **BY ME NOT WATCHING P0RN**... O_O
Just kidding, I am not joking. ( ͡° ͜ʖ ͡°). But anyway. One day I literally wanted to play with some mods, but I got an error. The error stated.. 
```
MultiMC version: 0.5.0-929
Minecraft folder is:
instances/MentricalCraftPLUS/minecraft
Java path is:
C:/Program Files (x86)/Java/jre1.8.0_121/bin/javaw.exe
Checking Java version...
Could not start java:
Error: Could not find or load main class JavaCheck
Check your MultiMC Java settings.
```
(sorry but I wanted to make less extra space to not piss you off)
So.. Me thinking I had my computer crying for a few days ago about the new java update. **SO BOOP** (*dramatic soundeffect*) I downloaded it. Restarted my Computer. Then Attempted it again. **SAME ERROR**.
So me being the smart person I am, I listened to the code and went to my multimc Setting. Auto-Detect. **NOT WORKING EITHER**. **THERE WAS NOTHING THERE**. *refreshes* **THEN I HAD A BREAKDOWN**. notreally. So then I went to my javaw.exe and manually directed the code to the stuff thing.. 
I just made a... `C:/Program Files (x86)/Java/jre1.8.0_121/bin/javaw.exe` **BOOM THERE YOU GO PATH I REMEMBER**. So I test that.... **DIDNT WORK EITHER**. It just popped up with the words of.. "`The specified java binary didn't work. You should use the auto-detect feature, or set the path to the java executable.`''
**WELL THATS HELPFUL.. OH YEA.**
so thats my problemo halps. :-[

What should happen:
------------------------------
I don't know what this means but. I think it means by. Wait lemme ask my Mum... She doesn't know. I think it meant to work? Thats whats meant to happen. should. 

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Babies? I'm literally not this smart. **I THOUGHT THE HTML5 LESSONS WHERE GREAT. BUT NOPE**.

Suspected cause:
---------------------------
I think the maybe most like cause stuff was um. I think its just how it was directed. 

Logs/Screenshots:
----------------------------
I don't have any photos, but I'm sure that's all.

Additional Info:
---------------------------
I am 50 years old, Hot and beautiful. Sorry 
I am a smartass.

**Comment by peterix** [comment]
Right. That's almost unreadable.

You are missing the `jars` folder of MultiMC or its contents. Or it's not readable for some reason.

Grab the current version from the website, and replace the `jars` folder in your current install:
![image](https://cloud.githubusercontent.com/assets/203326/22195614/32201b02-e149-11e6-97ed-724162a230e7.png)

**Comment by theuserscull** [comment]
thanks for the help, buddy person ting. :-]. But anyway yea my dumbass didn't had the slight of thinking about redownloading it. :/ But thanks! also by UNREADABLE you mean by my fantastic work with the words sentences.... Yea sorry :'(

**Comment by peterix** [comment]
Don't get me wrong, it's verbose enough and included everything needed to find out what's going on.

It's all the extra stuff ... I decided to **ADD FORMATTING TO IT**. *For **NO PARTICULAR REASON*** ;)

**Comment by DoDowapDooDowap** [comment]
Erm.... how can i say that.... I have the newest version of MultiMC and the jar folder, and IT KEEPS DOING THIS STUPID ERROR. I litteraly tried to the WHOLE DAY to fix this problem, THE WHOLE DAY. I posted in the official reddit, and NOTHING, I searched in google, NOTHING, Seriously, im with no patient so PLS HELP ME TO FIX THIS DANG PROBLEM. This is for @peterix 
![image](https://user-images.githubusercontent.com/126630912/222017496-f1ff1d91-d2e2-46ff-b428-544b4ad73adb.png)

**Comment by DoDowapDooDowap** [comment]
Erm.... how can i say that.... I have the newest version of MultiMC and the jar folder, and IT KEEPS DOING THIS STUPID ERROR. I litteraly tried to the WHOLE DAY to fix this problem, THE WHOLE DAY. I posted in the official reddit, and NOTHING, I searched in google, NOTHING, Seriously, im with no patient so PLS HELP ME TO FIX THIS DANG PROBLEM. This is for @peterix
![image](https://user-images.githubusercontent.com/126630912/222017766-a88bb325-f3d2-4105-bfc4-b66dbaf3a9e6.png)
